resolwe_bio/tests/processes/test_expression.py

```diff
@@ -66,13 +66,29 @@ def test_cuffnorm(self):
             inputs = {'src': 'cuffquant_2.cxb', 'source': "UCSC"}
             sample_2 = self.run_process("upload-cxb", inputs)
 
+            inputs = {'src': 'cuffquant_3.cxb', 'source': "UCSC"}
+            sample_3 = self.run_process("upload-cxb", inputs)
+
+            inputs = {'src': 'cuffquant_4.cxb', 'source': "UCSC"}
+            sample_4 = self.run_process("upload-cxb", inputs)
+
+            inputs = {'src': 'cuffquant_5.cxb', 'source': "UCSC"}
+            sample_5 = self.run_process("upload-cxb", inputs)
+
+            inputs = {'src': 'cuffquant_6.cxb', 'source': "UCSC"}
+            sample_6 = self.run_process("upload-cxb", inputs)
+
+            inputs = {'src': 'cuffquant_7.cxb', 'source': "UCSC"}
+            sample_7 = self.run_process("upload-cxb", inputs)
```

Comment:
What is the benefit of using so many samples? Keep in mind that tests should be small and fast.

Reply:
First we wish to have at least 3 groups as before we had only 2 groups and the tests did not catch some things that way. We also wanted to have at least 2 samples in each of these groups so we can see if the sorting works as it should. This is also the reason why we have 3 samples in one of the groups. We may be able to drop one sample from the test if that is preferred.